src/interfaces/navigation.ts

```diff
@@ -1,6 +1,6 @@
 import { RouteProp, NavigationProp, NavigationState } from '@react-navigation/native';
 
-export interface Navigation {
-  route: RouteProp<Record<string, object | undefined>, string>;
-  navigation: NavigationProp<Record<string, object | undefined>, string, NavigationState, {}, {}>;
+export interface Navigation<T extends Object | undefined> {
```

Comment:
`Object` is ok? Why not `object`?

Reply:
both seem to be working, what is the difference? 🤔